app/models/iterations/project_iterations.rb

```diff
@@ -23,31 +24,32 @@ def past_iterations
         PastIteration.new(
           start_date: start_at,
           end_date: end_at,
-          stories: stories_from(start_at, end_at),
-          iteration_number: iteration_number + 1
+          iteration_number: iteration_number + 1,
+          points: points_of_stories_between(start_at, end_at)
         )
       end
     end
 
     private
 
-    attr_reader :project
+    attr_reader :project, :stories
 
-    def stories_from(start_at, end_at)
+    def points_of_stories_between(start_at, end_at)
+      stories_between(start_at, end_at).to_a.map(&:estimate).compact.sum
+    end
+
+    def stories_between(start_at, end_at)
       stories.select do |story|
```

Comment:
What do you think about using a query here? Something like:
```ruby
stories.where(accepted_at: start_at..end_at)
```

Reply:
Here we have a problem using this query because I already make a query to bring all the `project.stories` from the database in line [49](https://github.com/Codeminer42/cm42-central/pull/407/files#diff-0648f597f84a74e685e5eabc0855e13cR49), this way it will make unnecessary consults all the time.

Thank you for the review :).